tests/Makefile

```diff
@@ -50,7 +50,9 @@ $(SUBST_FILES): %: %.in $(TOPDIR)/paths.mk
 check-syntax:
 	$(TOPDIR)/tests/syntax
 
-check: test-normal test-fltcmp test-boolfind test-bad-expected-results
+check-problems: test-normal test-fltcmp test-boolfind
```

Comment:
`check-problems` should also be added to the `.PHONY` targets as it's not a real file that gets created.